.gitattributes

```diff
@@ -0,0 +1,13 @@
+* text=auto
+
+/docs export-ignore
+/examples export-ignore
+/test export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.scrutinizer.yml export-ignore
+/.travis.yml export-ignore
+/travis.phpunit.xml.dist export-ignore
+/SECURITY.md export-ignore
+/changelog.md export-ignore
+/README.md export-ignore
```

Comment:
In pentest audits, presence of files like that is considered a risk due to information leakage. No need for them in production.